chpl/chpl-resources/src/main/resources/email.properties

```
@@ -331,8 +331,9 @@ complaintsReport.paragraph1.body=The report of all complaints is attached.
 #Developer Missing Attestations Change Request Email
 developer.missingAttestationChangeRequest.subject=Semi-annual Attestations Not Submitted
 developer.missingAttestationChangeRequest.salutation=Hello,
-developer.missingAttestationChangeRequest.paragraph1=According to our records, the Attestations for %s has not been submitted for the latest Attestations period. We are hereby requesting that this be submitted through the CHPL system as soon as possible. The following individuals are currently set up with access to complete and submit Attestations for your organization: %s.
-developer.missingAttestationChangeRequest.paragraph2=If there is another individual responsible for Attestations within your organization, please contact your ONC-ACB for further assistance.
+developer.missingAttestationChangeRequest.paragraph1=According to our records, the <a href='https://www.healthit.gov/condition-ccg/attestations'>Attestations Condition and Maintenance of Certification</a> for %s has not been submitted for the current Attestations period. As such, ONC is requesting that this be submitted through the CHPL system as soon as possible at <a href='https://chpl.healthit.gov/'>https://chpl.healthit.gov</a>.
```

```suggestion
developer.missingAttestationChangeRequest.paragraph1=According to our records, the <a href="https://www.healthit.gov/condition-ccg/attestations">Attestations Condition and Maintenance of Certification</a> for %s has not been submitted for the current Attestations period. As such, ONC is requesting that this be submitted through the CHPL system as soon as possible at <a href="https://chpl.healthit.gov/">https://chpl.healthit.gov</a>.
```

Our UI style guide says we should use double quotes for attribute values. It's not required here, but it would be more consistent with the other URLs in this file